components/admin-panel/sections/ForDevelopers.js

```diff
@@ -21,22 +22,28 @@ const ForDevelopers = ({ accountSlug }) => {
     return (
       <React.Fragment>
         <OAuthApplicationsList
-          accountSlug={accountSlug}
+          account={account}
           offset={query.offset ? parseInt(query.offset) : 0}
           onApplicationCreated={(app, account) => router.push(getOauthAppSettingsRoute(account, app))}
         />
-        <PersonalTokensList
-          accountSlug={accountSlug}
-          offset={query.offset ? parseInt(query.offset) : 0}
-          onPersonalTokenCreated={(app, account) => router.push(getPersonalTokenSettingsRoute(account, app))}
-        />
+        {/* Right now, it's USER with GraphQL v1 but why not make it compatible if we receive GraphQL v2 */}
+        {[CollectiveType.USER, CollectiveType.INDIVIDUAL].includes(account.type) && (
```

Comment:
There's a small helper for that

```suggestion
        {isIndividualAccount(account) && (
```